.github/ISSUE_TEMPLATE/onboarding.md

```diff
@@ -0,0 +1,25 @@
+---
+name: Onboarding
+about: 'Template for onboarding new developers '
+title: Onboarding to Azul
+labels: ''
+assignees: ''
+
+---
+
+# Onboarding
+To assist with getting onboarded to the project, review the following documents:
+- [ ] [Azul README.md](https://github.com/databiosphere/azul)
+
+- [ ] [Contributing Guidelines](https://github.com/DataBiosphere/azul/blob/develop/CONTRIBUTING.rst)
+
+Request access to cloud infrastructure and tools:
+- [ ] [UCSC-CGL Slack](ucsc-cgl.slack.com) through your Project Manager
+ 
+- [ ] Push access to the Azul Repository on [#team-boardwalk](https://ucsc-cgl.slack.com/archives/C705Y6G9Z)
+
+- [ ] Access to development CI/CD systems on [#team-boardwalk](https://ucsc-cgl.slack.com/archives/C705Y6G9Z)
+
+- [ ] [AWS Credentials](https://giwiki.gi.ucsc.edu/index.php/Overview_of_Getting_and_Using_an_AWS_IAM_Account)
+
+- [ ] [GCP Credentials](https://github.com/databiosphere/azul#232-google-credentials)
```

Comment:
```suggestion
- [ ] @hannes-ucsc: [GCP Credentials](https://github.com/databiosphere/azul#232-google-credentials)
```

Reply:
Changing the link to the google ocnsole url, moving the link above to the document section.